sdk/mixedreality/Azure.MixedReality.RemoteRendering/tests/Samples/RemoteRenderingConvertAssetSample.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.MixedReality.RemoteRendering.Tests.Samples
+{
+    // These tests assume that the storage account is accessible from the remote rendering account.
+    // See https://docs.microsoft.com/azure/remote-rendering/how-tos/create-an-account
+    // Since the roles can take a while to propagate, we do not live test these samples.
```

Comment:
is there any way we can programmatically test the propagation?

Reply:
I'm not sure, but I think it would be beyond the scope of a test for the data plane.